scheduler/src/cook/mesos/task.clj

```diff
@@ -42,7 +42,7 @@
         ;; If the custom-executor attr isn't set, we default to using a custom
         ;; executor in order to support jobs submitted before we added this field
         custom-executor? (:job/custom-executor job-ent true)
-        cook-executor? (and (not custom-executor?) (seq executor-config))
+        cook-executor? (and (not custom-executor?) (not container) (seq executor-config))
```

Comment:
Nitpick, can we break the `(not container)` onto a separate line and add a comment that we need to do additional work to add the cook executor to a container?